dlp/snippets/deid/deid_table_fpe.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package deid
+
+// [START dlp_deidentify_table_fpe]
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"io"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	"cloud.google.com/go/dlp/apiv2/dlppb"
+)
+
+// deidentifyTableFPE de-identifies table data with format preserving encryption.
+func deidentifyTableFPE(w io.Writer, projectID string, kmsKeyName, wrappedAesKey string) error {
+	// projectId := "your-project-id"
+	/* kmsKeyName := /* keyFileName :=  "projects/YOUR_PROJECT/"
```

Comment:
- Totally agreed @tritone, But previously got comments to minimalize the function arguments so that's the reason to keep it as commented part, apart from that in other samples this same practice was followed so as to be consistent across the repo used this format, But still let me know what should be done in this case, I will update the changes ASAP. Thank you for your efforts and suggestions.